packages/hooks/src/useOnKeyDown/useOnKeyDown.mdx

```diff
@@ -0,0 +1,36 @@
+---
+name: useOnKeyDown
+menu: Hooks
+route: /hooks/use-on-key-down
+---
+
+import { Playground } from "docz";
+import { useState } from "react";
+import { useOnKeyDown } from "./useOnKeyDown";
+import { Text } from "@jobber/components/Text";
+
+# UseOnKeyDown
+
+`useOnKeyDown` is an extremly simple hook that can be for adding the keydown event handler when the component is mounted and remove them when it unmount.
+
+`useOnKeyDown` should **only** only by used when making keyboard shortcut in any component.
+
+```tsx
+import { useOnKeyDown } from "@jobber/hooks";
+```
+
+
+<Playground>
+  {()=>{
+    const letterText = "Delete me by pressing Delete key on your keyboard.";
+    const letterText2 = "Space Enter and Shift + Escape will show here";
+    const [ text, setText ] = useState(letterText);
+    const [ textTwo, setTextTwo ] = useState(letterText2);
+    useOnKeyDown(() => setText("Deleted. Press Control + z for undo."), "Delete");
+    useOnKeyDown(() => setText(letterText), {key: "z", ctrlKey: true});
+    useOnKeyDown((e) => {
+        if(e.shiftKey) setTextTwo("You pressed Shift and "+e.key);
+        else setTextTwo("You pressed '"+e.key+"'");
+    }, [" ", "Enter", {key: "Escape", shiftKey: true} ]);
+    return (<Text>{text}<pre />{textTwo}</Text>);}}
```

Comment:
Looks like we have an extra `<pre />` here.

Reply:
Might also be nice to keep the instructions fixed and have a second line that displays the result of the key events.